build/esbuild-config.ts

```diff
@@ -24,4 +25,19 @@ export let esBuildContext = {
     ".svg": "dataurl",
   },
   outdir: "static/out",
+  define: createEnvDefines(["SUPABASE_URL", "SUPABASE_KEY"]),
```

Comment:
<img width="236" alt="Screenshot 2024-02-17 at 16 26 50" src="https://github.com/ubiquity/pay.ubq.fi/assets/4975670/2fc9ff8e-044e-446c-941e-8b8462283bc7">

Looks like we need to add this to our repo secrets?

Reply:
I added the necessary values and tried rerunning the CI but it failed again. Perhaps you can try pushing to force it to run again?

<img width="294" alt="Screenshot 2024-02-17 at 16 29 24" src="https://github.com/ubiquity/pay.ubq.fi/assets/4975670/7bd47a00-b8db-403f-9ef5-6f86e3222d25">

Reply:
My local works fine, I'll have a look at the logs. Also it uses the Supabase you shared to me